client/src/www/app/outline_server_repository/server.ts

```diff
@@ -34,7 +37,8 @@ export class OutlineServer implements Server {
     readonly accessKey: string,
     readonly type: ServerType,
     private _name: string,
-    private eventQueue: events.EventQueue
+    eventQueue: events.EventQueue,
+    localize: Localizer,
```

Comment:
I feel we need to clearly define layers and how they would interact with each other. For example, `outline_server_repository` might be a "model" layer, and `eventQueue` & `localize` are UI layers. Now it seems all of the components are mixed together.

Reply:
They are defined. The model has `Server` and `ServerRepository`, which are implementation-agnostic. The implementation here is part of the application layer.